scripts/config/webpack/config-client.js

```diff
@@ -231,6 +232,10 @@ module.exports = ({ minify } = {}) => {
                 allChunks: true,
                 disable: isDev,
             }),
+            !isDev && minify && new CompressionPlugin({
+                asset: '[path].gz',
+                minRatio: 1,
```

Comment:
This is outdated, when I was putting all the compressed files in the `build/compressed` folder, I increased the threshold so no files were missing. This is no longer required since `gzipStatic` serves the regular file if the compressed file doesn't exist.
I will remove both custom options.